testsuite/E70.py

```diff
@@ -12,6 +12,8 @@
 del a[:]; a.append(42);
 #: E704:1:1
 def f(x): return 2
+#: E704:1:1
+async def f(x): return 2
```

Comment:
I've fixed this case as well as two others where 'async  def' was not being detected